### PR TITLE
Added June 2021 Donation Information

### PR DESCRIPTION
**Pull request by RiseCardano**

### CODE REVIEW

**Review by kiriakos — APPROVED** [review]
LGTM

File: RISE/donations.yml [hunk 1]
```diff
@@ -67,4 +67,28 @@
   timestamp: 1622456220
   files:
     - RISE/SevaUkMay2021Receipt.PNG
+  
+- ticker: RISE
+  announcement_url: &ann https://twitter.com/RiseCardano/status/1410227893947490314
+  recipient: 
+    name: Water Aid UK
```
Comment (kiriakos) [inline RISE/donations.yml:74]: can you add the `contact` field please? Can be an email.

File: RISE/donations.yml [hunk 2]
```diff
@@ -67,4 +67,28 @@
...
+  ada: 542.815
+  usd: 721.96
+  timestamp: 1625046780
+  files:
+    - RISE/WaterAidReceiptJune2021.PNG
+
+- ticker: RISE
+  announcement_url: &ann https://twitter.com/RiseCardano/status/1423651080618094592
+  recipient: 
+    name: Save The Children
```
Comment (kiriakos) [inline RISE/donations.yml:86]: Here too, can you add the `contact` field please? Can be an email.

**Review by kiriakos — CHANGES_REQUESTED** [review]

**Comment by aloopmarin007** [comment]
Please create a new updated PR , Thank you, We will close this one since it's out of date.

**Pull request closed without merging**